yt_dlp/extractor/yahoo.py

```diff
@@ -177,15 +177,16 @@ class YahooIE(InfoExtractor):
     }]
 
     def _extract_yahoo_video(self, video_id, country):
-        video = self._download_json(
-            f'https://{country}.yahoo.com/_td/api/resource/VideoService.videos;view=full;video_ids=["{video_id}"]',
-            video_id, 'Downloading video JSON metadata')[0]
-        title = video['title']
+        data = self._download_json(
+            f'https://video-api.yql.yahoo.com/v1/video/sapi/streams/{video_id}',
+            video_id, 'Downloading video metadata', fatal=False)
```

Comment:
why `fatal=False` ? this should be fatal
```suggestion
            video_id, 'Downloading video JSON metadata')
```